BaseSpace.SDK/ServiceModels/Request/GetUserPermissionRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿
+
+namespace Illumina.BaseSpace.SDK.ServiceModels
+{
+    public class GetUserPermissionRequest : AbstractResourceRequest<GetUserPermissionResponse>
```

Comment:
Hi,

Do you want me to rename the UserPermission class to something like AccessTokenDetails  as well?

[DataContract]
public class UserPermission : AbstractResource

Thanks,
Van

From: maximmass [mailto:notifications@github.com]
Sent: Tuesday, August 4, 2015 2:07 PM
To: basespace/basespace-csharp-sdk <basespace-csharp-sdk@noreply.github.com>
Cc: Le-Pham, Van <vlepham@illumina.com>
Subject: Re: [basespace-csharp-sdk] Get current user permission and add more return properties to SearchResult (#41)


In BaseSpace.SDK/ServiceModels/Request/GetUserPermissionRequest.cs<https://github.com/basespace/basespace-csharp-sdk/pull/41#discussion_r36242010>:

> @@ -0,0 +1,14 @@

> +﻿

> +

> +namespace Illumina.BaseSpace.SDK.ServiceModels

> +{

> +    public class GetUserPermissionRequest : AbstractResourceRequest<GetUserPermissionResponse>

perhaps rename this and the client method to something like GetAccessTokenDetails ?

—
Reply to this email directly or view it on GitHub<https://github.com/basespace/basespace-csharp-sdk/pull/41/files#r36242010>.